test/absinthe/phase/parse_test.exs

```diff
@@ -82,14 +82,27 @@ defmodule Absinthe.Phase.ParseTest do
   test "handle parse error when column not available" do
     assert {:error, bp} = Absinthe.Phase.Parse.run(@graphql, jump_phases: false)
 
-    assert [
-             %Absinthe.Phase.Error{
-               extra: %{},
-               locations: [%{column: 0, line: 4}],
-               message: "syntax error before: ",
-               phase: Absinthe.Phase.Parse
-             }
-           ] == bp.execution.validation_errors
+    otp = :erlang.system_info(:otp_release)
+
+    if otp == '24' do
+      assert [
+               %Absinthe.Phase.Error{
+                 extra: %{},
+                 locations: [%{column: 3, line: 4}],
```

Comment:
weird behavior difference where otp 24 now has column info

Reply:
Yeah this is expected, this is a fine way to handle it.